entries/innerHeight.xml

```diff
@@ -9,7 +9,7 @@
   </signature>
   <desc>Get the current computed height for the first element in the set of matched elements, including padding but not border.</desc>
   <longdesc>
-    <p>This method returns the height of the element, including top and bottom padding, in pixels.</p>
+    <p>This method returns the height of the element, including top and bottom padding, in pixels. If called on an empty set of elements, returns <code>undefined</code> (<code>null</code> before jQuery 2.2.0).</p>
```

Comment:
Hmm, https://github.com/jquery/jquery/blame/master/src/dimensions.js#L50 must be lying somehow (or I'm misreading it). But you're right. Fixed.